### ISSUE TITLE
reopen: People Picker dropdown list is not being announced in Android

### ISSUE THREAD

**Issue by yumikohey** [OfficeDev/office-ui-fabric-react#10412]
This is repro in Android app and Win32 apps.

#### Codepen demonstrating the issue
https://developer.microsoft.com/en-us/fabric#/controls/web/peoplepicker
#### Actual behavior:
When users navigate with a physical keyboard up and down, voiceover is silent.

#### Expected behavior:
When users navigate with a physical keyboard up and down on the dropdown available list, voiceover should announce the name of the person.

**Comment by dzearing** [comment]
VoiceOver has nothing to do with Win32. @yumikohey Could you be more clear here? Which browser are you talking about, on which platform/hardware, using which reader?
